prototype/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}.py

```diff
@@ -59,10 +59,5 @@ def workbench_scenarios():
         """A canned scenario for display in the workbench."""
         return [
             ("{{cookiecutter.class_name}}",
-             """<vertical_demo>
-                <{{cookiecutter.short_name|lower}}/>
-                <{{cookiecutter.short_name|lower}}/>
-                <{{cookiecutter.short_name|lower}}/>
-                </vertical_demo>
-             """),
+            """<{{cookiecutter.short_name|lower}}/>"""),
```

Comment:
I would argue that handling a common error case should be done in standard tests rather than in the default view. Defaults shouldn't be used for testing, but for easy-onboarding.